share/plugins/courtesy_accidentals/addCourtesyAccidentals.qml

```diff
@@ -18,298 +18,131 @@
 //==============================================
 
 import QtQuick 2.0
-import QtQuick.Dialogs 1.2
 import MuseScore 3.0
+import Qt.labs.settings 1.0
```

Comment:
Please remove the use of this plugin (it does not come with Qt6, which we plan to migrate to soon). `Settings` has been moved from it to `MuseScore`, i.e. nothing additional needs to be done.